### PR TITLE
Update for A20

### PR DESCRIPTION
**Pull request by s0600204**
Update mod for the recent A20 release of 0AD.

* White-space cleanup
* Resolving units' build and structures' train lists to use inheritance from parent templates
* Change units' sound-groups to use new folder structure

No balancing has been carried out.

### CODE REVIEW

File: simulation/templates/template_unit_support_female_citizen.xml [hunk 1]
```diff
@@ -20,16 +20,13 @@
...
-		</BonusStructure>
-
-</Bonuses>
+      <Bonuses>
+        <BonusStructure>
+          <Classes>Structure</Classes>
+          <Multiplier>7.0</Multiplier>
+        </BonusStructure>
+      </Bonuses>
+      <RestrictedClasses datatype="tokens">Infantry Cavalry Champion Structure CitizenSoldier</RestrictedClasses>
```
Comment (niektb) [inline simulation/templates/template_unit_support_female_citizen.xml]: Do you restrict the ability to attack here?

File: simulation/templates/template_unit_support_female_citizen.xml [hunk 2]
```diff
@@ -45,11 +42,18 @@
...
       structures/{civ}_market
       structures/{civ}_civil_centre
+      structures/{civ}_outpost
+      structures/{civ}_wallset_wood
+      structures/{civ}_barracks
+      structures/{civ}_blacksmith
+      structures/{civ}_defense_tower
+      structures/{civ}_wallset_stone
+      structures/{civ}_fortress
+      structures/{civ}_wonder
```
Comment (niektb) [inline simulation/templates/template_unit_support_female_citizen.xml]: Are these buildings meant to be built with female citizens?

Comment (s0600204) [inline simulation/templates/template_unit_support_female_citizen.xml]: @LordGood had added those buildings to the build lists of the respective `{earth|pega|uni}_support_female_citizen.xml` files, so as unusual as it seems, it appears so.

File: simulation/templates/template_unit_support_female_citizen.xml (hunk 1 above)
Comment (s0600204) [inline simulation/templates/template_unit_support_female_citizen.xml]: That's... I'm... not sure why I put that restriction there. :confused:

File: simulation/templates/template_unit_support_female_citizen.xml (hunk 2 above)
Comment (niektb) [inline simulation/templates/template_unit_support_female_citizen.xml]: Alright

**Pull request merged**